### PR TITLE
Feature/form validation

### PR DESCRIPTION
**Pull request by rs-mgb**

### CODE REVIEW

File: src/es/components/molecules/simpleForm/SimpleForm.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { Shadow } from '../../prototypes/Shadow.js'
  *
  * }
  */
-export default class SimpleForm extends Shadow() {
+export default class SimpleForm extends Validation() {
```
Comment (Weedshaker) [inline src/es/components/molecules/simpleForm/SimpleForm.js:25]: I would handle this analog the [Wrapper](https://github.com/mits-gossau/web-components-toolbox/blob/master/src/es/components/organisms/wrapper/Wrapper.js#L21), except this extend has Zero-Risk to effect the SimpleForm.

First:
`export const SimpleForm = (ChosenHTMLElement = Shadow) => class SimpleForm extends ChosenHTMLElement {`

Second:
`export default class SimpleFormValidation extends SimpleForm(Validation())`

Then you have a new Class SimpleFormValidation and you can use both independent. What do you think?

File: src/es/components/prototypes/Validation.js [hunk 2]
```diff
@@ -0,0 +1,379 @@
+// @ts-check
+
+import { Shadow } from './Shadow.js'
+
+export const Validation = (ChosenClass = Shadow()) => class Validation extends ChosenClass {
+  /**
+   * Creates an instance of Shadow. The constructor will be called for every custom element using this class when initially created.
```
Comment (Weedshaker) [inline src/es/components/prototypes/Validation.js:7]: A bit more description would be appreciated.

**Review by Weedshaker — CHANGES_REQUESTED** [review]
I would separate SimpleForm from Validation and create a SimpleFormValidation extending SimpleForm

File: src/es/components/prototypes/Validation.js (hunk 2 above)
Comment (rs-mgb) [reply src/es/components/prototypes/Validation.js:7]: Added short description of Validation file

File: src/es/components/molecules/simpleForm/SimpleForm.js (hunk 1 above)
Comment (rs-mgb) [reply src/es/components/molecules/simpleForm/SimpleForm.js]: Fixed at Pair-programming together

**Review by Weedshaker — APPROVED** [review]

**Pull request merged**